### DISCUSSION TITLE
Support AMP (Accelerated Mobile Pages) Features

### DISCUSSION THREAD

**Discussion by radityaarhya** [adityatelange/hugo-PaperMod#1135] [Ideas]
It would be nice if this **awesome theme** has a feature like this

1. The theme is the same like the current version
2. But the AMP is available for the mobile user

or

1. All the Theme Feature is ready for AMP

Don't know if this possible or not

**Comment by adityatelange** [comment]
Oh we don't need AMP with PaperMod, it's already fast enough to beat the FCP and FID. 😉
https://pagespeed.web.dev/report?url=https://adityatelange.github.io/hugo-PaperMod/
